src/main/scala/mesosphere/marathon/core/task/termination/TaskKillConfig.scala

```diff
@@ -0,0 +1,41 @@
+package mesosphere.marathon.core.task.termination
+
+import org.rogach.scallop.ScallopConf
+import scala.concurrent.duration._
+
+import scala.concurrent.duration.FiniteDuration
+
+trait TaskKillConfig extends ScallopConf {
+  //scalastyle:off magic.number
+
+  private[this] lazy val _killChunkSize = opt[Int](
+    "kill_chunk_size",
+    descr = "INTERNAL TUNING PARAMETER: " +
+      "The maximum number of concurrently processed kills",
+    noshort = true,
+    hidden = true,
+    default = Some(100)
+  )
+
+  private[this] lazy val _killRetryTimeout = opt[Long](
+    "kill_retry_timeout",
+    descr = "INTERNAL TUNING PARAMETER: " +
+      "The timeout after which a task kill will be retried.",
+    noshort = true,
+    hidden = true,
+    default = Some(10000L) // 10 seconds
```

Comment:
ack; copied and pasted from another ScallopConf trait and didn't even change the value there ;)